templates/ruby/lib/container/client.rb.twig

```diff
@@ -100,10 +100,10 @@ module {{spec.title | caseUcfirst}}
             when nil   then ''
             else
                 post_body = []
-                if(key.to_s == "file")
-                    post_body << "Content-Disposition: form-data; name=\"#{key}\"; filename=\"#{File.basename(value)}\"\r\n"
-                    post_body << "Content-Type: #{MIME::Types.type_for(value)}\r\n\r\n"
-                    post_body << File.read(value)
+                if(value.instance_of? AppwriteFile)
```

Comment:
Avoid the word `Appwrite` inside the templates. You can use a Twig param to produce in the generated code.

Reply:
Because we already use {{ spec.title | caseDash }} as a namespace we might be able to use just `File` for the class name.